test/tools/libtesteth/Options.h

```diff
@@ -44,7 +44,7 @@ class Options
     bool exectimelog = false; ///< Print execution time for each test suite
     std::string rCurrentTestSuite; ///< Remember test suite before boost overwrite (for random tests)
     bool statediff = false;///< Fill full post state in General tests
-    bool fulloutput = false;///< Replace large output to just it's length
+    bool fullstate = false;///< Replace large state output to just it's length or hash
```

Comment:
it's never length, right?

Reply:
right. it used to be. but perhaps hash is better

Reply:
If block has incorrect uncles, it could be consensus issue with equal state roots.

Anyway, here delete the mention of length, if it's not true anymore.